packages/ml-kit/README.md

```diff
@@ -0,0 +1,156 @@
+|----------------|-----------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `acceleration` | <code>DelegateType &vert; number</code> | Which type of acceleration to use when running the model. It can take the values `DelegateType.GPU` (GPU acceleration), `DelegateType.NNAPI` ([Android Neural Networks API](https://developer.android.com/ndk/guides/neuralnetworks) acceleration) or a number indicating the quantity of threads to use. |
+
+- `ModelNameResolver: () => ModelOptions`
+
+Useful to change the options used by the model of a task at runtime.
+
+**Tasks output events:**
+
+- [`{classificationAim}Predicted`](#events)
+- [`{regressionAim}Predicted`](#events)
+
+> Example usage in the application task graph:
+> 
+> ```typescript
+> on('inputDataForHumanActivity', run('humanActivityClassificationUsingCNN'));
+> on('humanActivityPredicted', run('writeRecords'));
+> 
+> on('inputDataStressLevel', run('stressLevelRegression'));
+> on('stressLevelPredicted', run('writeRecords'));
+> ```
+> 
+> **Note**: To use the `writeRecords` task, the persistence package must be installed and configured. See [persistence package docs](https://github.com/GeoTecINIT/awarns-framework/blob/main/packages/persistence/README.md).
+
+### Events
+
+| Name                           | Payload                             | Description |
+|--------------------------------|-------------------------------------|-------------|
+| `{classificationAim}Predicted` | [`Classification`](#classification) |             |
+| `{regressionAim}Predicted`     | [`Regression`](#regression)         |             |
+
+
+### Records
+
+#### Classification
+
+| Property               | Type                                            | Description                                                |
+|------------------------|-------------------------------------------------|------------------------------------------------------------|
+| `id`                   | `string`                                        | Record's unique id.                                        |
+| `type`                 | `string`                                        | Always `classification-prediction`.                        |
+| `change`               | `Change`                                        | Always `NONE`.                                             |
+| `timestamp`            | `Date`                                          | The local time when the classification record was created. |
```

Comment:
How about: "The local time when the model predicted the classification result?" The "was created" thing seems a bit ambiguous to me.